interface/login/login.php

```diff
@@ -107,6 +111,10 @@
 
     <title><?php echo text($openemr_name) . " " . xlt('Login'); ?></title>
 
+    <?php
+    if ($GLOBALS['google_signin_enabled']) { ?>
```

Comment:
if google-signin-client_id needs to be unempty for this to work then would also ensure this in above conditional

Reply:
Fixed